src/core/Table/Table.tsx

```diff
@@ -595,7 +595,7 @@ export const Table = <
   );
 
   const virtualizedItemRenderer = React.useCallback(
-    (index: number) => getPreparedRow(page[index]),
+    (index: number) => getPreparedRow(page[index], index),
```

Comment:
Looks weird, when you pass instance and its index.. Could just pass index then and get the instance in `getPreparedRow`, but I guess minor and less changes :D

Reply:
Changed